### PR TITLE
Adjust "custom block text style" addon description

### PR DESCRIPTION
**Pull request by MaterArc**
### Changes

Tiny grammar fix to make it sound more fluent.

### Reason for changes

Fluency

### CODE REVIEW

File: addons/custom-block-text/addon.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Customizable block text style",
-  "description": "Allows you to adjust the size and thickness of block text or add a shadow to make it easier to read.",
+  "description": "Allows you to adjust the size and font weight of block text or add a shadow to make it easier to read.",
```
Comment (DNin01) [inline addons/custom-block-text/addon.json:3]: "Size and font weight" sounds slightly inconsistent. I wonder if saying "font size and weight" or just "size and weight" would be better. I don't know, what do some other folks think?

Wait for some conversation before changing anything.

**Comment by WorldLanguages** [comment]
Isn't "thickness" good enough?

**Comment by just-a-hriday** [comment]
> Isn't "thickness" good enough?

Yeah, it sounds better to me too.

**Comment by MaterArc** [comment]
Thickness just sounds off to me, that could just be me. It sounds choppy. It took me more than once reading the description to understand what they were actually trying to say. Again, this could just be a me thing...

**Comment by BroJac5246** [comment]
> Thickness just sounds off to me, that could just be me. It sounds choppy. It took me more than once reading the description to understand what they were actually trying to say. Again, this could just be a me thing...

I'm not a fan of "thickness," either. But do people know what font weight is?

**Comment by WorldLanguages** [comment]
Font weight is probably the right term, but it's hard to tell how many users will understand what it means. As a non-native English speaker "thickness" sounds more intuitive, I basically had to memorize the "font-weight" CSS property due to its frequency.

**Comment by DNin01** [comment]
Maybe it would be better to say "use bold text" instead of "adjust the font weight".

**Comment by WorldLanguages** [comment]
> Maybe it would be better to say "use bold text" instead of "adjust the font weight".

Yeah, that would reflect the current customization options.

File: addons/custom-block-text/addon.json (hunk 1 above)
Comment (DNin01) [inline addons/custom-block-text/addon.json:3]:
```suggestion
  "description": "Lets you customize the style of text on blocks, including adjusting the size, using bold text, or adding a shadow.",
```

**Comment by MaterArc** [comment]
Thanks, @DNin01 ! It looks much better now :)

**Comment by MaterArc** [comment]
Nope, I think it's merge-ready

**Pull request merged**